_posts/2019-01-23-dhcpcd-7.1.0.markdown

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: "dhcpcd>=7.1.0 update may require manual intervention"
+comments: false
+---
+
+Starting with 7.1.0 the [dhcpcd(8)](https://man.voidlinux.org/dhcpcd.8) package is dropping the 10-wpa_supplicant hook
+in favour of setting CONFIG_MATCH_IFACE in [wpa_supplicant(8)](https://man.voidlinux.org/wpa_supplicnt.8), as suggested by upstream.
```

Comment:
`. Instead of the hook, upstream wants users to set CONFIG_MATCH_IFACE in [wpa_supplicant(8)](https://man.voidlinux.org/wpa_supplicnt.8)`

(maybe this should be a doc link instead? `wpa_supplicant.conf`?